examples/hitl/rearrange_v2/config/language_rearrange.yaml

```diff
@@ -43,23 +43,17 @@ habitat:
     task_spec: pddl/multi_agent_tidy_house
     pddl_domain_def: fp
     actions:
-      base_velocity:
+      agent_0_base_velocity:
```

Comment:
Should be uniformized with the other velocities in this config.

I deployed the following:
```
lin_speed: 10.0
ang_speed: 300
```